site/en/tutorials/distribute/custom_training.ipynb

```diff
@@ -676,7 +676,7 @@
         "\n",
         "1. [Distributed training guide](../../guide/distributed_training)\n",
         "2. [DenseNet](https://github.com/tensorflow/examples/blob/master/tensorflow_examples/models/densenet/distributed_train.py) example using `MirroredStrategy`.\n",
-        "1. [BERT](https://github.com/tensorflow/models/blob/master/official/nlp/bert/run_classifier.py) example trained using `MirroredStrategy` and `TPUStrategy`.\n",
+        "1. [BERT](https://github.com/tensorflow/models/blob/v2.8.0/official/nlp/bert/run_classifier.py) example trained using `MirroredStrategy` and `TPUStrategy`.\n",
```

Comment:
Let's use an example from the main repository:

```suggestion
        "1. [BERT](https://github.com/tensorflow/models/commits/master/official/legacy/bert/run_classifier.py) example trained using `MirroredStrategy` and `TPUStrategy`.\n",
```

Following @lehougoogle 's commit https://github.com/tensorflow/models/commit/27fb855b027ead16d2616dcb59c67409a2176b7f